### DISCUSSION TITLE
Collapsible Parent Folders for Profile List

### DISCUSSION THREAD

**Discussion by KurtisLiggett** [snappysnappydog/Simple-IP-Config#127] [Ideas]
Thinking about changing the profile list to something like below, to better organize large lists.

![image](https://user-images.githubusercontent.com/29669372/161329015-5c30d0c0-38aa-4679-b095-eb03d078a244.png)

**Comment by boardnick0815** [comment]
Hi,

i really would appreciate this nice feature!

Aaron

**Comment by blackcrack** [comment]
\<loadprogram>
for let show the actual workgroup: "wmic computersystem get workgroup" => \<take the 2nd line/output>=> put into the Textfild as default  
Workgroup : [_<ins>Michashomesmbnet</ins>_________][set]   // the SetButton is for post the textfild to wmic for change the workgroup

hmm.. could be pretty well for SMB Workgroups .. perhaps couple this with it ?

School Workgroup=[Schoolgroup__________]  //default entry is readout from definition configuration
Work1 Workgroup=[Officegroupinvoice____] //default entry is readout from definition configuration
Work2 Workgroup=[OfficegroupinPersonal_] //default entry is readout from definition configuration
Work3 Workgroup=[Officegroupinworkshop] //default entry is readout from definition configuration
Work4 Workgroup=[Officegroupingarage___] //default entry is readout from definition configuration

just as a idea.. /2Cent

**Comment by jaschkopf** [comment]
Hello. Are there any news on this idea? It's exactly the feature I'm missing. I need to sort many many fixed IP Addresses in groups to keep a good overview.
